tensorflow_app/views/import_graphdef.py

```diff
@@ -55,17 +56,23 @@ def get_padding(node, layer):
     pad_top = pad_along_height / 2
     pad_left = pad_along_width / 2
     '''
-    if node.type == "Conv3D":
+
+    if node.type == "Conv3D" or node.type == "MaxPool3D" or node.type == "AvgPool3D":
```

Comment:
`node.type in {"Conv3D", "MaxPool3D", "AvgPool3D"}` will make it cleaner and you can extend it without too much or conditions whenever you need to add more support of 3D layers.